controller/main/main.ino

```diff
@@ -104,3 +135,85 @@ void loop()
 
   sendMessageToFrontend();
 }
+
+//Actuator Control Functions:
+
+//gets the current actuator distance in mm:
+int getActuatorDistance(int actuator)
+{
+  int scalingFactor=(maxActuatorValue[actuator-1]-minActuatorValue[actuator-1])/50;
```

Comment:
You may already know this, but this is integer division. It won't round the number, but truncate everything past the decimal.

I.e: `137/50 = 2`